src/test/java/org/jenkinsci/plugins/ewm/TestUtil.java

```diff
@@ -47,4 +62,28 @@ public int compare(Disk disk1, Disk disk2) {
             }
         });
     }
+
+    public static void addExternalWorkspaceNodeProperty(Node node, String diskPoolRefId, DiskNode... diskNodes) {
```

Comment:
This method is used only for testing purposes. 
Is there any need to restrict it in this case?